services/hangar/hangar-base.js

```diff
@@ -0,0 +1,37 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const description = `
+<p><a href="https://hangar.minecraft.tools/">Hangar</a> is a plugin repository for the Paper, Waterfall and Folia platforms.</p>`
```

Comment:
What's this host btw?

Reply:
Which one?

Reply:
`hangar.minecraft.tools`

Reply:
Oh, I don't even know how that ended up in there, fixing right now